generate.py

```diff
@@ -1,21 +1,21 @@
 import requests
 import os
 
```

Comment:
On my local branch, this is actually recursive so, you no longer have to input manual files to reload. I'll be pushing that up soon before I accept this PR.

Reply:
Does that mean a solution for #29

Reply:
Correct.